v2/internal/reconcilers/arm/arm_client_cache_test.go

```diff
@@ -199,19 +302,19 @@ func newSecret(name string, namespace string) *v1.Secret {
 	}
 }
 
-func newResourceGroup(name string, namespace string) *resources.ResourceGroup {
+func newResourceGroup(namespace string) *resources.ResourceGroup {
```

Comment:
Why hard code the name of the resource group?

Reply:
Its to avoid linter complaining about the same name being passed every time